python_modules/dagster/dagster/_core/definitions/partition_mapping.py

```diff
@@ -191,6 +197,9 @@ def get_downstream_partitions_for_partitions(
     ) -> PartitionsSubset:
         raise NotImplementedError()
 
+    def __str__(self) -> str:
```

Comment:
I think calling this `description` would be better, because it allows `__str__` to show a more Dagster-developer-accessible version of the class's representation (with the class name and attributes).